src/Google.Pubsub.V1/PublisherClient.cs

```diff
@@ -448,12 +557,31 @@ public sealed partial class PublisherClientImpl : PublisherClient
             );
 
         private readonly ClientHelper _clientHelper;
+        private readonly ApiCall<Topic, Topic> _createTopic;
+        private readonly ApiCall<PublishRequest, PublishResponse> _publish;
+        private readonly ApiCall<GetTopicRequest, Topic> _getTopic;
+        private readonly ApiCall<ListTopicsRequest, ListTopicsResponse> _listTopics;
+        private readonly ApiCall<ListTopicSubscriptionsRequest, ListTopicSubscriptionsResponse> _listTopicSubscriptions;
+        private readonly ApiCall<DeleteTopicRequest, Empty> _deleteTopic;
 
         public PublisherClientImpl(Publisher.IPublisherClient grpcClient, PublisherSettings settings)
         {
-            this.GrpcClient = grpcClient;
+            GrpcClient = grpcClient;
             PublisherSettings effectiveSettings = settings ?? PublisherSettings.GetDefault();
+            IClock effectiveClock = effectiveSettings.Clock ?? SystemClock.Instance;
             _clientHelper = new ClientHelper(effectiveSettings);
+            _createTopic = _clientHelper.BuildApiCall<Topic, Topic>(GrpcClient.CreateTopicAsync, GrpcClient.CreateTopic)
```

Comment:
Fine for now, but I wonder whether BuildApiCall should have an overload taking a RetrySettings. Then it could just be

```csharp
_createTopic = _clientHelper.BuildApiCall<Topic, Topic>(GrpcClient.CreateTopicAsync, GrpcClient.CreateTopic, effectiveSettings.CreateTopicRetry);